src/app/invest/invest.component.html

```diff
@@ -103,53 +107,44 @@ <h1 class="app-basic-title">{{ states.project.name }}</h1>
                     </div>
-                    <div class="estimated__invested-label col-lg-12">
+                    <div class="estimated__invested-label">
                         <p>{{ investmentDetails?.amountInvested | currencyDefault }} invested so far </p>
                     </div>
 
                 <div class="row">
-                    <div class="col-lg">
+                    <div class="col-lg-4">
                         <div class="estimated__return-header">
-                            <div class="estimated__header-label col-lg"><p>Return over 1 year</p></div>
+                            <div class="estimated__header-label"><p>Return over 1 year</p></div>
                         </div>
                         <div class="return__amounts">
-                            <div class="estimated__amount-label col-lg"><p>{{ oneYearReturn }}</p>
+                            <div class="estimated__amount-label"><p>{{ oneYearReturn }}</p>
                             </div>
                         </div>
                     </div>
-                    <div class="col-lg">
+                    <div class="col-lg-4">
                         <div class="estimated__return-header">
-                            <div class="estimated__header-label col-lg"><p>5 Years</p></div>
+                            <div class="estimated__header-label"><p>5 Years</p></div>
                         </div>
                         <div class="return__amounts">
-                            <div class="estimated__amount-label col-lg"><p>{{ fiveYearsReturn }}</p>
+                            <div class="estimated__amount-label"><p>{{ fiveYearsReturn }}</p>
                             </div>
                         </div>
                     </div>
-                    <div class="col-lg">
+                    <div class="col-lg-4">
                         <div class="estimated__return-header">
-                            <div class="estimated__header-label col-lg"><p>10 Years</p></div>
+                            <div class="estimated__header-label"><p>10 Years</p></div>
                         </div>
                         <div class="return__amounts">
-                            <div class="estimated__amount-label col-lg"><p>{{ tenYearsReturn }}</p>
+                            <div class="estimated__amount-label"><p>{{ tenYearsReturn }}</p>
                             </div>
                         </div>
                     </div>
                 </div>
             </div>
         </div>
-
-        <div class="funds__bottom">
-            <div class="dropdown-divider"></div>
-            <div class="row row-cols-2">
-                <p class="right__available">Available funds</p>
-                <h2>{{ investmentDetails.walletBalance | currencyDefault }}</h2>
-                <a class="right__link" routerLink="/dash/wallet">Open wallet</a>
-            </div>
-        </div>
```

Comment:
If we show funds only in calculator element, it won't be visible on use cases where we don't have calculator shown (when ROI is 0). I implemented new funds section for mobile screens (lines 44-52)